### DISCUSSION TITLE
Threshold labels should be displayed in the selected unit

### DISCUSSION THREAD

**Discussion by JeanFred** [grafana/grafana#36974] [Dashboards, Panels & Visualizations]
- [X] Feature request

When using the singlestat panel with a gauge with threshold labels, I would expect the labels displayed in the unit selected for the singlestat.

See this screenshot for an extreme example: I have data about time in the range of years ; but it is retrieved as a Statsd timer, so in milliseconds. I would want « 3 » (for 3 years) to be displayed instead of the equivalent in milliseconds
![screenshot from 2016-06-13 15-21-36](https://cloud.githubusercontent.com/assets/364381/16010771/929c7df6-317a-11e6-8b4e-554d910be2b8.png)

Please include this information:
- Grafana 3.0.4
- Graphite datasource
- Ubuntu 14.04

**Comment by Serphentas** [comment]
Any updates on this ? Would gladly welcome some tips on making the required changes and potentially a PR.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82946. Feel free to continue the discussion around this there. Thank you!
